api/src/main/java/org/pmiops/workbench/db/model/DbWorkspaceOperation.java

```diff
@@ -0,0 +1,117 @@
+package org.pmiops.workbench.db.model;
+
+import java.sql.Timestamp;
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+
+@Entity
+@Table(name = "workspace_operation")
+public class DbWorkspaceOperation {
```

Comment:
Ah, if the timestamps weren't working - you may need this annotation here:

```
@EntityListeners(AuditingEntityListener.class)
```

Annoying piece of boilerplate... I know. Perhaps there is a way to elevate this to a central config instead..

Reply:
aha, thanks!